core-bundle/contao/templates/twig/content_element/text.html.twig

```diff
@@ -21,7 +23,10 @@
 
     {# Richtext #}
     {% block text %}
-        {% set text_attributes = attrs(text_attributes|default).addClass('rte') %}
+        {% set text_attributes = attrs()
+            .mergeWith(text_attributes|default)
+            .addClass('rte')
```

Comment:
Should this come before the `mergeWith()` so it can be unset? The same goes for `.setIfExists('nonce', csp_nonce('script-src'))` or `.set('type', 'button')`.

Reply:
I think `.mergeWith(…)` should go to the end of the modifications in all base templates. That was the idea behind moving to `mergeWith()` AFAIR.